### ISSUE TITLE
Support sass-embedded

### ISSUE THREAD

**Issue by ahmadhabannakehkaseya** [elycruz/rollup-plugin-sass#152]
### A little background:
`sass-embedded` is a package maintained by the same team behind `sass` package and is based on the Dart Sass implementation, they should have the same API except that `sass` is pure javascript while `sass-embedded` is a wrapper over native Dart Sass executable, the latter becomes faster for large projects because of that.

We need to support that runtime, it's already supported by `sass-loader` for webpack:
https://github.com/webpack-contrib/sass-loader/releases/tag/v15.0.0

The `sass-embedded` package only comes with modern JS API not the legacy sass API, not sure what this plugin is based on, but it might need some work and maybe some breaking changes are expected for users.

Currently, I tried passing `import * as sassRuntime from 'sass-embedded'` to `runtime` option (I use ES Modules in rollup config, should be the same as `require.resolve('sass-embedded')` in CommonJS) but it failed with `cannot read properties of undefined (reading 'then')`

**Comment by elycruz** [comment]
Right, I hear you @ahmadhabannakehkaseya - We may need to support modern sass importer functions.  

Heads-up: The issue that was causing the "reading `then` on `undefined`" you mentioned above, is fixed now - So if you can give your approach another go it's possible it may work out of the box now (new plugin version is [v1.13.2](https://github.com/elycruz/rollup-plugin-sass/releases/tag/v1.13.2)).

**Comment by ahmadhabannakehkaseya** [comment]
@elycruz Thank you, I tried `1.13.2`, while the error disappears another issue happens: the css file that should be the output of sass building is empty.
